CMakeLists.txt

```diff
@@ -284,13 +284,7 @@ string( REPLACE ";" " " ALL_LIBS "${ALL_LIBS}" )
 
 # all includes
 set( ALL_INCLUDES_tmp
-  "${PROJECT_SOURCE_DIR}/libnestutil"
-  "${PROJECT_BINARY_DIR}/libnestutil"
-  "${PROJECT_SOURCE_DIR}/librandom"
-  "${PROJECT_SOURCE_DIR}/sli"
-  "${PROJECT_SOURCE_DIR}/topology"
-  "${PROJECT_SOURCE_DIR}/nestkernel"
-  "${PROJECT_SOURCE_DIR}/nest"
+  "${CMAKE_INSTALL_FULL_INCLUDEDIR}/nest"
```

Comment:
Why remove these? Shouldn't a variable starting with ``ALL_INCLUDES`` indeed contain all include folders? ^^

Reply:
The `ALL_INCLUDES` variable is only used in [`nest-config`](https://github.com/nest/nest-simulator/blob/master/extras/nest-config.in#L61). Here we expect to get locations of installed header files. The `${PROJECT_SOURCE_DIR}/...` are only available, as long as the NEST sources are there, but (here) we install all headers to `"${CMAKE_INSTALL_FULL_INCLUDEDIR}/nest"`, hence remove all source related paths and have only the install path.